definitions/variable/CO2.yaml

```
@@ -0,0 +1,7 @@
+# This yaml file adds needed variables for E3M´s model data upload
+
+- Capacity|Carbon Capture|Direct Air Capture:
+    description: Installed (available) capacity to extract CO2 from the atmosphere
+      using Direct Air Capture (DAC)
+    unit: Mt CO2/
+    
```

Removing the "/" from the unit and the whitespaces in the last line
```suggestion
    unit: Mt CO2

```

---

I deleted accidentally part of the units.